scripts/generateExamplesTests.js

```diff
@@ -86,27 +86,34 @@ function fixupExamples(examples) {
       const steps = [];
 
       for (const step of job.steps) {
-        if (step.uses && step.uses.startsWith('infracost/actions/comment')) {
-          const path = step.with.path;
+        if (step.name && step.name.toLowerCase() === 'post infracost comment') {
           const goldenFilePath = `./testdata/${jobKey}_comment_golden.md`;
+          const commentArgs = step.run
+            .replace(/\\/g, '')
```

Comment:
We can change the behavior here like this:
```js
.replace(/--behavior update/, '--behavior new')
```

Reply:
I'm gonna keep it with `behavior update` as it's just a dry run

Reply:
Sorry, I didn't mention it, but this comment relates to https://github.com/infracost/actions/pull/57#discussion_r825739550 - to avoid making API requests when running CI.

Reply:
If you don't mind I'd rather keep the `update` as I value actually testing the API calls are 👌  at this point.

Reply:
No objections here. Personally I would add unit tests to the commenting code instead of making real API requests, but of course it's out of scope of the PR.